src/boards_pinout.h

```diff
@@ -296,4 +297,4 @@
     #define RADIO_TXEN          14
 #endif
 
-#endif
\ No newline at end of file
+#endif
```

Comment:
Please remove this hunk.

Reply:
wha does this means?

Reply:
@richonguzman he unintentionally added a newline at the end of the file. His editor did in fact. I'm asking to remove the hunk introducing this change :)